undefined_syms.txt

```diff
@@ -1682,6 +1682,17 @@ D_060119B0 = 0x060119B0;
 D_0600DC40 = 0x0600DC40;
 D_0600DB60 = 0x0600DB60;
 
+// z_en_ta
+D_060017E8 = 0x060017E8;
```

Comment:
these are probably in the repo already with the latest undefined syms import? doesnt matter all too much since its temporary anyway, but yea